lib/lotus/model.rb

```diff
@@ -16,7 +17,7 @@ module Model
     # @since 0.1.0
     #
     # @see Lotus::Repository.update
```

Comment:
@karimmtarek Instead of requiring a separated file with that exception, can you please define it here? Thank you!